applications/models.py

```diff
@@ -362,6 +363,36 @@ def can_be_cancelled(self):
     def can_confirm(self):
         return self.status in [APP_INVITED, APP_LAST_REMIDER]
 
+    def serialize(self):
+        return {
+            'user': {
+                'name': self.user.name,
+                'email': self.user.email,
+                'is': {
+                    'active': self.user.is_active,
+                    'volunteer': self.user.is_volunteer,
+                    'organizer': self.user.is_organizer,
+                    'director': self.user.is_director,
+                    'sponsor': self.user.is_sponsor,
+                    'admin': self.user.is_admin,
+                    'hardwareAdmin': self.user.is_hardware_admin,
+                },
+                'application': {
+                    # University
+                    'graduationYear': self.graduation_year,
+                    'classStatus': self.class_status,
+                    'university': self.university,
+                    'degree': self.degree,
+                    # Info for swag and food
+                    'diet': self.diet,
+                    'otherDiet': self.other_diet,
+                    'tshirtSize': self.tshirt_size,
+                    # Info for hardware
+                    'hardware': self.hardware,
+                }
```

Comment:
Overall, the font size when displaying the badge info is decently small on desktop and on mobile some of the info is clipped off at the bottom. I would recommend upping the font size a bit and making that part scrollable (or removing some info we don't necessarily need to display).